db/schema.rb

```diff
@@ -308,7 +308,6 @@
     t.string "slug"
     t.string "status", default: "active"
     t.datetime "updated_at", null: false
-    t.index ["slug"], name: "index_chat_channels_on_slug", unique: true
```

Comment:
Hmmm is this line unintentionally committed? 🤨

Reply:
Nope. This field should long been gone but keeps coming back.

Reply:
👍👍 I’ve been seeing it show up intermittently but I thought it was just my local DB. Good riddance! :)

Reply:
@vaidehijoshi Here's a recent PR that also removed it and we should be careful to make sure it eventually stays gone: https://github.com/thepracticaldev/dev.to/pull/7528